peer/peer.go

```diff
@@ -316,15 +315,19 @@ func MaintainConnections() {
 // PeerInfoHandler will handle the response to a PeerInfo request by attempting
 // to establish connections with all new peers in the given response Resource.
 func PeerInfoHandler(res *msg.Response) {
-	peers := res.Resource.([]string)
+	peers := res.Resource.([]interface{})
```

Comment:
Right but we're type casting them. Why can't we type cast it directly to `[]string` like before rather than type casting each element individually?